sealtk/core/test/ScalarFilterModel.cpp

```diff
@@ -0,0 +1,203 @@
+/* This file is part of SEAL-TK, and is distributed under the OSI-approved BSD
+ * 3-Clause License. See top-level LICENSE file or
+ * https://github.com/Kitware/seal-tk/blob/master/LICENSE for details. */
+
+#include <sealtk/core/ScalarFilterModel.hpp>
+
+#include <sealtk/core/AbstractItemModel.hpp>
+#include <sealtk/core/DataModelTypes.hpp>
+
+#include <vital/types/timestamp.h>
+
+#include <vital/range/iota.h>
+
+#include <QSet>
+
+#include <QtTest>
+
+namespace kvr = kwiver::vital::range;
+
+using time_us_t = kwiver::vital::timestamp::time_t;
+
+// ----------------------------------------------------------------------------
+template<> char* QTest::toString<QSet<int>>(QSet<int> const& set)
```

Comment:
`error: specialization of 'template<class T> char* QTest::toString(const T&)' in different namespace ... from definition of 'template<class T> char* QTest::toString(const T&)' [-fpermissive]`

Reply:
Wrapping it in `namespace QTest{` and removing `QTest::` fixes the error for me.

Reply:
Looks like this is a [GCC bug](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=56480). Still, it's an easy fix, so we should fix it anyway.